src/inc/corinfo.h

```diff
@@ -695,6 +695,9 @@ enum CorInfoHelpFunc
     CORINFO_HELP_THROW_ARGUMENTOUTOFRANGEEXCEPTION, // throw ArgumentOutOfRangeException
 #endif
 
+    CORINFO_HELP_JIT_PINVOKE_BEGIN, // Transition to preemptive mode before a P/Invoke, frame is the first argument
```

Comment:
I believe these are used only by Core RT right?  In such a case should these be constrained to be available when compiling Core RT?